pages/linux/genisoimage.md

```diff
@@ -0,0 +1,12 @@
+# genisoimage
+
+> Genisoimage is a pre-mastering program to generate ISO9660/Joliet/HFS hybrid filesystems.
+> More information: <https://manpages.debian.org/latest/genisoimage/genisoimage.1.en.html>.
+
+- Create an ISO image:
```

Comment:
Since we pass a directory argument, I think it might help to mention it in the description too. Something like

> Create an image on a specific directory:

Btw, what happens if the directory argument isn't passed?

Reply:
Good question!
If the directory argument is not passed, the program will display a message asking for the directory path.

Reply:
Perhaps this is more descriptive, I'll change it: Create an ISO image from the given source directory

Reply:
Resolving as this was applied in a5dcedff0d0663c99d1ce0bb92e62b4899f34237 and bdb809118f49f17c38f125a0104b156f7a9b5d54.